dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops

```diff
@@ -712,6 +712,10 @@ error.invalidPhaseName=CWWKE0954E: The "{0}" checkpoint phase specified is empty
 error.invalidPhaseName.explanation=An unknown phase was specified as the value of the checkpoint option.
 error.invalidPhaseName.useraction=Review the documentation to determine valid phase names. Reissue the command using a valid checkpoint phase.
 
+error.set.securitymanager.jdk18=CWWKE0955E: Java Security was requested in bootstrapping.properties while using JDK "{0}", however this option is no longer valid when using Java 18 and later.
```

Comment:
The JEP is titled "Deprecate the Security Manager for Removal".  On the other hand there is the `java.security` package which contains lots of other security related stuff.  A subset of that package contains stuff related to the Security Manager, like Permission classes, but doesn't includ the actual `java.lang.SecurityManager` class.

I think we need to be cautious with a message that makes it sound like anything with security in the package/title in Java is being deprecated.  I suggest a message like:

The property "websphere.java.security" was set in the bootstrap.properties file while using Java "{0}".  This property enables the Java Security Manager and is no longer valid when using Java 18 or later.